packages/next/src/client/components/app-router-headers.ts

```diff
@@ -11,6 +11,7 @@ export const NEXT_ROUTER_PREFETCH_HEADER = 'Next-Router-Prefetch' as const
 // be merged into a single enum.
 export const NEXT_ROUTER_SEGMENT_PREFETCH_HEADER =
   'Next-Router-Segment-Prefetch' as const
+export const NEXT_ROUTER_STALE_TIME_HEADER = 'Next-Router-Stale-Time' as const
```

Comment:
nit: this group of headers are the request headers. The section at the bottom (postponed and prerender) are the response headers

Reply:
In that case I'll use the `x-nextjs-` convention too